ambry-router/src/test/java/com/github/ambry/router/GetBlobOperationTest.java

```diff
@@ -1992,7 +2033,7 @@ public void testDecompressContent() throws Exception {
 
     // Generate the test compressed buffer.
     ByteBuffer sourceBuffer =
-        ByteBuffer.wrap("Compression unit test to test compression.".getBytes(StandardCharsets.UTF_8));
+        ByteBuffer.wrap("Compression unit test to test compression." .getBytes(StandardCharsets.UTF_8));
```

Comment:
Is the space a typo?

Reply:
updated, the formatter is messing up my PR here.